proto/scanner/api/v1/node_inventory_service.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
+
+option go_package = "scannerV1";
+
+option java_package = "io.stackrox.proto.api.scanner.v1";
+
+import weak "google/api/annotations.proto";
+
+import "scanner/api/v1/note.proto";
+import "scanner/api/v1/component.proto";
+
+package scannerV1;
+
+message GetNodeInventoryResponse {
+    string                      node_name   = 1;
+    Components                  components  = 2;
+    repeated Note               notes       = 3;
+}
+
+message GetNodeInventoryRequest {}
+
+// NodeInventoryService is used in Secured Clusters to fetch information from Nodes and communicate with other
+// Secured Cluster components, like the compliance container in Collector.
+service NodeInventoryService {
```

Comment:
(Answering to Note to reviewers) I like this name

Reply:
+1 - I like the name as well